### ISSUE TITLE
Removing duplicated items

### ISSUE THREAD

**Issue by peonso** [hjnilsson/rme#130]
`Cleanup...` menu option could do this as well, something like this maybe? https://gist.github.com/peonso/835501dd76bf856aaf434200dfe9736b/327e182ca539c746af4d2fd965141188ccc63287#file-convertmap-lua-L1125-L1139

**Comment by Mignari** [comment]
you gave me an idea. I will try to implement lua on RME to externalize some functions. :+1:

**Comment by peonso** [comment]
I'm in a quest to be able to convert some maps. I started with
- https://otland.net/threads/rme-replacing-items.245343/

found those 2 threads
- https://otland.net/threads/saving-map-in-game-creates-a-new-otbm.141986/
- https://otland.net/threads/savemap-function-saves-it-to-a-new-otbm.193781/

made this
- https://gist.github.com/peonso/835501dd76bf856aaf434200dfe9736b

that is working, but is slow as fuck and very limited, seeked some help
- https://otland.net/threads/need-more-efficiency-in-a-script-challenge-for-knowledgeable-test-your-skills.245622/

found this also
- https://github.com/hjnilsson/rme/blob/46e21c425639f5acbfd9ca0e6ed0103664261a71/source/map.cpp#L148
- https://github.com/hjnilsson/rme/blob/46e21c425639f5acbfd9ca0e6ed0103664261a71/source/map.cpp#L76
- https://github.com/hjnilsson/rme/blob/46e21c425639f5acbfd9ca0e6ed0103664261a71/source/templatemap81.cpp

it works? My knowledge is limited.

**Comment by Mignari** [comment]
sorry, I commented something unrelated to the issue. I don't know if the convert method works.
but you want to replace or remove?

**Comment by peonso** [comment]
You got it right, I myself that brought another point about what you said, just explaining why I came up with lua to solve my problem. I want to convert 10.xx maps to 7.6 based on a conversion table I made. The lua function does it, bit it has a lot of things missing and is slow, I want to know of this part of not used code can be implemented and would solve my issue. This just adds up with the "remove duplicate items" thing, I added it to the function.
